service/src/main/java/uk/gov/hmcts/reform/aac/model/ChangeOrganisationRequest.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.aac.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import uk.gov.hmcts.reform.ccd.model.Organisation;
+import uk.gov.hmcts.reform.fpl.model.common.dynamic.DynamicList;
+
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ChangeOrganisationRequest {
+
+    @JsonProperty("OrganisationToAdd")
+    private Organisation organisationToAdd;
+
+    @Setter
+    @JsonProperty("OrganisationToRemove")
+    private Organisation organisationToRemove;
+
+    @JsonProperty("CaseRoleId")
+    private DynamicList caseRoleId;
+
+    @JsonProperty("RequestTimestamp")
+    @JsonSerialize(using = LocalDateTimeSerializer.class)
+    private LocalDateTime requestTimestamp;
+
+    @JsonProperty("ApprovalStatus")
+    private String approvalStatus;
```

Comment:
Do we want this to be an enum that serialises to the appropriate value based on the [following definition](https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1380222099#id-2.CCDTeamAssignAccesstoaCasepart2DeliveryScope-NewChangeOrganisationRequestComplexFieldStandardBaseType)?

|Field|Type|Validation|comment|
|---|---|---|---|
|ApprovalStatus|Number|NULLABLE|0 = ‘Not considered’, 1 = ‘Approved’, 2 = ‘Rejected’.  Note that a value of 2 will always be ephemeral until the rejection is processed.|

Reply:
done